recipe/meta.yaml

```
@@ -1,25 +1,33 @@
-{% set version = "2.1.2" %}
+{% set version = "2.2.1" %}
 
 package:
   name: mgwr
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/m/mgwr/mgwr-{{ version }}.tar.gz
-  sha256: cc5e845e70f169a0eb9aaf2cb27391cecb6cff27ca4916f7e9f7bba19338d26d
+  sha256: 8b00ba992fd85c9528327eae88e104e072939d0d5203b72c513ef29d5c400ff7
+  patches:
+    - fix_imports.patch
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<310]
```

https://github.com/pysal/mgwr/blob/v2.2.1/setup.py#L40
```suggestion
  skip: true  # [py<35]
```

---

No, it depends on libpysal >=4.10 that requires py<310 at least,  otherwise libpysal 4.5.1 (noarch) can be picked up with an incompatible scipy constraint.

---

Added comment to the recipe